validation-test/Sanitizers/tsan-inout.swift

```diff
@@ -23,8 +28,14 @@ var gInThread2: () -> () = { }
 // Spawn two threads, run the two passed in closures simultaneously, and
 // join them.
 func testRace(name: String, thread inThread1: @escaping () -> (), thread inThread2: @escaping () -> ()) {
+#if canImport(Darwin)
   var thread1: pthread_t?
   var thread2: pthread_t?
+#else
+  var thread1: pthread_t = 0
+  var thread2: pthread_t = 0
+  var t : pthread_t = 0
+#endif
```

Comment:
Windows doesn't have pthreads

Reply:
This test didn't run on Windows before and will not run after my change.